src/components/EntityNodeCard.tsx

```diff
@@ -289,12 +292,13 @@ export default memo(({ node }: { node: EntityNode }) => {
             )}
             {settings.showFace && faceImage ? (
               <img
-                alt={faceImage?.alt}
+                alt=""
```

Comment:
mmmh, I fear a google penalization, but ok

Reply:
It's a super long text, better nothing in case the image doesn't load